### PR TITLE
update to phpunit 9.5

### PR DESCRIPTION
**Pull request by lukejolliffe**
# Purpose

Update PHPUnit
Fixes UML-2667

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by lukejolliffe** [comment]
Screenshot 2022-09-14 at 11 12 51
Screenshot 2022-09-14 at 11 13 00

File: service-front/app/test/CommonTest/Form/AbstractFormTest.php [hunk 1]
```diff
@@ -20,7 +23,7 @@ protected function setUp(): void
     {
         $guardProphecy = $this->prophesize(CsrfGuardInterface::class);
 
-        $this->form = new class('testConcreteForm', $guardProphecy->reveal()) extends AbstractForm {
+        $this->form = new class ('testConcreteForm', $guardProphecy->reveal()) extends AbstractForm {
```
Comment (cooperaj) [inline service-front/app/test/CommonTest/Form/AbstractFormTest.php:26]:
```suggestion
        $this->form = new class('testConcreteForm', $guardProphecy->reveal()) extends AbstractForm {
```
Did your IDE suggest this?

File: service-front/app/test/CommonTest/Middleware/Authentication/AuthenticationMiddlewareTest.php [hunk 2]
```diff
@@ -9,6 +9,7 @@
 use Laminas\Stratigility\MiddlewarePipeInterface;
 use Mezzio\Authentication\AuthenticationMiddleware as MezzioAuthenticationMiddleware;
 use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
```
Comment (cooperaj) [inline service-front/app/test/CommonTest/Middleware/Authentication/AuthenticationMiddlewareTest.php:12]: Added trait but not used?

File: service-front/app/test/CommonTest/Service/Features/FeatureEnabledTest.php [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 use Common\Service\Features\FeatureEnabled;
 use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
```
Comment (cooperaj) [inline service-front/app/test/CommonTest/Service/Features/FeatureEnabledTest.php:9]: Not used

File: service-front/app/test/CommonTest/Service/Url/UrlValidityCheckServiceTest.php [hunk 4]
```diff
@@ -11,30 +11,22 @@
...
+
+    private ServerRequestInterface|ObjectProphecy $serverRequestFactoryProphecy;
+
+    protected RouterInterface|ObjectProphecy $routerProphecy;
+
+    protected ServerRequestFactoryInterface|ObjectProphecy $serverRequestInterfaceProphecy;
+
+    private UrlHelper|ObjectProphecy $urlHelperProphecy;
 
     private $locale;
```
Comment (cooperaj) [inline service-front/app/test/CommonTest/Service/Url/UrlValidityCheckServiceTest.php:31]:
```suggestion
    private ServerRequestInterface|ObjectProphecy $serverRequestFactoryProphecy;
    protected RouterInterface|ObjectProphecy $routerProphecy;
    protected ServerRequestFactoryInterface|ObjectProphecy $serverRequestInterfaceProphecy;
    private UrlHelper|ObjectProphecy $urlHelperProphecy;
    private $locale;
```

It's ok to cuddle the properties

**Review by cooperaj — CHANGES_REQUESTED** [review]

File: service-front/app/test/CommonTest/Form/AbstractFormTest.php (hunk 1 above)
Comment (lukejolliffe) [reply service-front/app/test/CommonTest/Form/AbstractFormTest.php:26]: Psalm did I think

Comment (lukejolliffe) [reply service-front/app/test/CommonTest/Form/AbstractFormTest.php:26]: Will double check later

Comment (cooperaj) [reply service-front/app/test/CommonTest/Form/AbstractFormTest.php:26]: If it's been suggested by a linter then I guess it's what we should be doing.

Comment (lukejolliffe) [reply service-front/app/test/CommonTest/Form/AbstractFormTest.php:26]: Screenshot 2022-09-14 at 14 40 33
It's in my IDE but also was raised by phpcs in the github action D:

File: service-front/app/test/CommonTest/Middleware/Authentication/AuthenticationMiddlewareTest.php (hunk 2 above)
Comment (lukejolliffe) [reply service-front/app/test/CommonTest/Middleware/Authentication/AuthenticationMiddlewareTest.php:12]: I think I've put the trait into these and then removed it when I realised it wasn't used

File: service-front/app/test/CommonTest/Form/AbstractFormTest.php (hunk 1 above)
Comment (cooperaj) [reply service-front/app/test/CommonTest/Form/AbstractFormTest.php:26]: spotted. it. it is a PSR12 thing. go figure.

**Review by cooperaj — APPROVED** [review]

**Pull request merged**